File: 프리_이상남/6주차 미션/Chapter 6. Java/src/exception.java
import java.util.Scanner;

public class exception {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        int num = scanner.nextInt();
        scanner.close();

        try {
            int result = 10 / num;
            System.out.println("Result: " + result);
        } catch (ArithmeticException e) {
            System.out.println("Cannot divide by zero");
        }
    }
}

File: 프리_이상남/6주차 미션/Chapter 6. JavaScript/exception.js
const readline = require('readline');
 
// 인터페이스 객체를 만들자.
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});
try {
  const result = num / 0;
  console.log("Result: " + result);
} catch (error) {
  console.log("Cannot divide by zero");
}